examples/MBQCvqe.py

```diff
@@ -81,22 +103,32 @@ def simulate_mbqc(self, params, backend="tensornetwork"):
 
     # %%
     # Function to compute the energy
-    def compute_energy(self, params):
+    def compute_energy(self, params: Iterable[float]):
         # Simulate the MBQC circuit using tensor network backend
         tn = self.simulate_mbqc(params, backend="tensornetwork")
         # Compute the expectation value using MBQCTensornet.expectation_value
         energy = tn.expectation_value(self.hamiltonian, qubit_indices=range(self.n_qubits))
         return energy
 
 
+class MBQCVQE_WithPlaceholders(MBQCVQE):
```

Comment:
Please use PEP8-compliant name (maybe without `_` ?)

Reply:
Fixed in 8e9fb19.